src/components/RevokeButton/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+
+import CodeRevokeModal from '../../containers/CodeRevokeModal';
+
+class RevokeButton extends React.Component {
+  state = {
```

Comment:
seems like there is some shared boilerplate with the RemindButton that could potentially be consolidated?

Reply:
Agreed. Working on consolidating these with a new component (something like `ActionButtonWithModal` perhaps) using a HOC or render props :)